### DISCUSSION TITLE
Raspberry Pi 5 + google coral 𝗢𝗥 N100 miniPC + openVINO for Frigate NVR?

### DISCUSSION THREAD

**Discussion by lukasig** [blakeblackshear/frigate#9722] [Ask A Question]
[this post on REDDIT](https://www.reddit.com/r/frigate_nvr/comments/1al9uwb/raspberry_pi_5_google_coral_vs_n100_minipc/)

Hi

I am deciding on new hardware to run 4 IP cameras (2K) using frigate.
The considered options are:

1. Raspberry Pi 5 + NVMe hat + coral + 256 SSD (80 + 20 + 25 + 20 = 145 USD approx.)

2. [N100 dual NICs mini PC](https://vi.aliexpress.com/item/1005004893120495.html) with openVINO - 256NVMe 16GB RAM (180 USD approx.)

After watching [Jeff Geerling's video](https://www.youtube.com/watch?v=jjzvh-bfV-E) and the inference speed in [this video](https://www.youtube.com/watch?v=so8_h0EKPWU), I am wondering if the N100 would perform well enough running 4 cameras using openVINO instead of google Coral.

Considering that when running a raspberry Pi with coral a powered USB hub is also needed + a USB-SATA adapter, the cost of either setups seem to be pretty much the same.

The advantage of the N100 mini PC is that it comes with dual NICs that allow you to setup a dedicated private network for your cameras where they can be blocked from accessing the internet. Being an x86 processor it can also run other software (like ms. Windows) if needed.

Power consumption I expect to be similar. Probably the RPi5 setup would be around 10W and the N100 mini PC around 15W (just a wild guess)

What other things I should be taking into consideration before deciding?

Thanks

**Comment by NickM-27** [comment]
I'd definitely recommend the Intel route given those options

**Reply by yousaf465** [reply]
Running Frigate 0.14 on beelink EQ12 N100 with two cameras. Speed around 14ms.
Don't have a coral yet.

**Comment by lukasig** [comment]
Just to come up with an update.
I did go the Intel route for two systems.

1. Using an Intel N97 processor (very similar to N100) using openVINO running 4 cameras at 2K resolution. Inference speed around 10ms.
2. Using an Intel i3 N305 processor and a google coral installed on the wifi card m.2 slot, for now running only 3 cameras at 2K resolution. Inference speed around 8ms.

Running on a raspberry pi I think the recommend hardware would be the rpi4 + a usb coral anyway, because the rpi5 does not have h264 hardware decoding.

**Reply by yousaf465** [reply]
coral will not do the hardware decoding

**Reply by lukasig** [reply]
> coral will not do the hardware decoding

Sorry, that is not what I intended to say. You need a coral no matter which rpi you choose. But since rpi4 has h264 decoding and rpi5 only has h265, most would make better use of rpi4.
